README.md

```diff
@@ -29,38 +35,11 @@ beta ![status](https://img.shields.io/badge/status-beta-yellow) )
 stable ![status](https://img.shields.io/badge/status-stable-brightgreen) )
 -->
 
-For more info about the features, please refer to our [documentation](https://lifecycle.keptn.sh/docs/).
-
-## Watch the KubeCon 2022 Detroit Demo
-
-Click to watch it on YouTube:
-
-[![Keptn Lifecycle Toolkit in a Nutshell](https://img.youtube.com/vi/K-cvnZ8EtGc/0.jpg)](https://www.youtube.com/watch?v=K-cvnZ8EtGc)
-
-## Deploy the latest release
-
-### Known Limitations
-
-- Kubernetes >=1.24 is needed to deploy the Lifecycle Toolkit
-- The Lifecycle Toolkit is currently not compatible with [vcluster](https://github.com/loft-sh/vcluster)
-
-### Installation
-
-Use the following command sequence to install the latest release of the Keptn Lifecycle Toolkit:
-
-```shell
-helm repo add klt https://charts.lifecycle.keptn.sh
-helm repo update
-helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
-```
-
-For installing the Lifecycle Toolkit via manifests use:
-
-<!---x-release-please-start-version-->
-
-```shell
-kubectl apply -f https://github.com/keptn/lifecycle-toolkit/releases/download/v0.8.0/manifest.yaml
-```
+See the Keptn Lifecycle Toolkit
+[documentation](https://lifecycle.keptn.sh/docs/)
+for information about installing and implementing KLT
+as well as migrating to KLT from
+[Keptn v1](https://keptn.sh/docs/).
 
 <!---x-release-please-end-->
```

Comment:
since you removed the version numbers in the text block above, you also don't need the release please annotations anymore here.